contracts/0.4.24/lib/PositiveTokenRebaseLimiter.sol

```diff
@@ -0,0 +1,153 @@
+// SPDX-FileCopyrightText: 2023 Lido <info@lido.fi>
+
+// SPDX-License-Identifier: GPL-3.0
+
+/* See contracts/COMPILERS.md */
+pragma solidity 0.4.24;
+
+import {SafeMath} from "@aragon/os/contracts/lib/math/SafeMath.sol";
+
+import {Math256} from "../../common/lib/Math256.sol";
```

Comment:
Let's rename to something more meaningful or have overloaded func inside the func for uint256 and int256

Reply:
done